src/core/database/database.service.ts

```diff
@@ -71,6 +71,40 @@ export const permission = (property: string, baseNode: string) => {
   ];
 };
 
+export const permissions = (baseNode: string, properties: string[]) => {
+  return properties.flatMap((property) => permission(property, baseNode));
+};
+
+export const property = (
+  prop: string,
+  value: any | null,
+  baseNode: string,
+  propVar = prop,
+  propLabel?: string
+) => {
+  const createdAt = DateTime.local().toISO();
```

Comment:
```suggestion
  const createdAt = DateTime.local();
```
We want to use native dates in DB instead of strings